management/user.go

```diff
@@ -343,3 +383,28 @@ func (m *UserManager) Blocks(id string, opts ...RequestOption) ([]*UserBlock, er
 func (m *UserManager) Unblock(id string, opts ...RequestOption) error {
 	return m.Request("DELETE", m.URI("user-blocks", id), nil, opts...)
 }
+
+// RegenerateRecoveryCode removes the current multi-factor authentication recovery code and generates a new one.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/post_recovery_code_regeneration
+func (m *UserManager) RegenerateRecoveryCode(id string) (*UserRecoveryCode, error) {
+	r := new(UserRecoveryCode)
+	err := m.Request("POST", m.URI("users", id, "recovery-code-regeneration"), r)
+	return r, err
+}
+
+// Enrollments retrieves all Guardian enrollments for a user.
+//
+// See: https://auth0.com/docs/api/management/v2#!/Users/get_enrollments
+func (m *UserManager) Enrollments(id string) (enrols []*UserEnrollment, err error) {
+	err = m.Request("POST", m.URI("users", id, "enrollments"), &enrols)
+	return enrols, err
+}
+
+// ListAuthenticators retrieves all authenticators for a user.
+//
+// It's an undocumented API ¯\_(ツ)_/¯
```

Comment:
I've asked about that API in Auth0 community so I think you can use the following link at least for now.
https://community.auth0.com/t/when-will-api-v2-users-userid-authenticators-be-documented/52722

Reply:
Hah, interesting that you found this PR while looking for ways to manage MFA.

I truly don't understand how that endpoint is still undocumented: it was recommended to me by an Auth0 engineer (via their support) since a while now, and it's a  very valuable endpoint.